README.md

```diff
@@ -0,0 +1,2 @@
+#Queueify
+Queueify returns queued strings in an simple and easy way
```

Comment:
I would replace the description like: 
"Queueify allows to create queues of asynchronous tasks. The API is promised-based"